portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +27,46 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  private static final Gson gson = new Gson();
```

Comment:
nit: we should change this to "GSON" (all caps variable name) since it's constant:
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
done.